packages/perseus/src/widgets/interactive-graphs/graphs/components/arrowhead.test.tsx

```diff
@@ -0,0 +1,22 @@
+import {render} from "@testing-library/react";
+import React from "react";
+
+import {Arrowhead} from "./arrowhead";
+
+describe("Arrowhead", () => {
+    it("Uses provided color to fill the arrow", () => {
+        const {container} = render(
+            <Arrowhead tip={[0, 0]} angle={0} color="green" />,
+        );
+        // eslint-disable-next-line testing-library/no-container,testing-library/no-node-access
+        const svgArrow = container.querySelector("path");
+        expect(svgArrow?.getAttribute("style")).toContain("stroke: green");
+    });
+
+    it("Inherits the color of the parent element when no color is specified", () => {
+        const {container} = render(<Arrowhead tip={[0, 0]} angle={0} />);
+        // eslint-disable-next-line testing-library/no-container,testing-library/no-node-access
+        const svgArrow = container.querySelector("path");
+        expect(svgArrow?.getAttribute("style")).toContain("stroke: inherit");
+    });
+});
```

Comment:
Do you think it would make sense to add a test for the SVG's `transform` property given a certain `angle` here?

Reply:
Philosophically, it shouldn't matter how the arrow is rotated. So if we decide in the future to do that differently, we shouldn't have to update our tests. I think that if we want to verify that the arrow is rotated properly, then we should look into setting up visual regression tests (i.e. screen snapshots).